code/game/machinery/porta_turret/portable_turret.dm

```diff
@@ -670,7 +686,11 @@
 	nonlethal_projectile_sound = 'sound/weapons/taser2.ogg'
 	lethal_projectile = /obj/item/projectile/beam/laser
 	lethal_projectile_sound = 'sound/weapons/laser.ogg'
-	desc = "An energy blaster auto-turret."
+	desc = "An energy blaster auto-turret. Use a welder to fix."
+	explosion_block = 50
+	max_integrity = 260
+	integrity_failure = 20
+
```

Comment:
This will affect other energy turrets outside the pirate ship ones, please make a new subtype for them space seafolks.